BrightID/src/components/Onboarding/OnboardScreen.tsx

```diff
@@ -145,68 +198,133 @@ export const Onboard = () => {
             </TouchableOpacity>
           </View>
         </View>
-        <View style={styles.versionInfoContainer}>
+        {/* <View style={styles.versionInfoContainer}>
```

Comment:
Why is the versioninfo removed? This is important information when users are reporting problems.